translate/convert/test_po2prop.py

```diff
@@ -247,6 +247,20 @@ def test_merging_untranslated_multiline(self):
         print(propfile)
         assert propfile == ''  # We drop the key
 
+    def test_merging_untranslated_multiline2(self):
+        """check merging untranslated entries with multiline values"""
+        posource = '''#: legal_text_and_links3\nmsgid "By using {{clientShortname}} you agree to the {{terms_of_use}} and {{privacy_notice}}."\nmsgstr ""\n'''
```

Comment:
IMHO it is readable to do use of multiline strings and avoid having everything on a single line which uses `\n`. So:

```python
        posource = '''
#: legal_text_and_links3
msgid "By using {{clientShortname}} you agree to the {{terms_of_use}} and {{privacy_notice}}."
msgstr ""
'''
```